examples/medplum-photon-integration/src/bots/handle-order-event.ts

```diff
@@ -18,37 +18,36 @@ import {
   OrderEventType,
   OrderFulfillmentData,
   OrderReroutedData,
+  PhotonEvent,
   PhotonPrescription,
-  PhotonWebhook,
 } from '../photon-types';
 import { NEUTRON_HEALTH, NEUTRON_HEALTH_WEBHOOKS } from './system-strings';
-import { getExistingMedicationRequest, handlePhotonAuth, photonGraphqlFetch, verifyEvent } from './utils';
+import { checkForDuplicateEvent, getExistingMedicationRequest, handlePhotonAuth, photonGraphqlFetch } from './utils';
 
 export async function handler(
   medplum: MedplumClient,
-  event: BotEvent<PhotonWebhook>
+  event: BotEvent<PhotonEvent>
 ): Promise<MedicationRequest | undefined> {
-  const webhook = event.input;
+  const body = event.input as OrderEvent;
   // You will need to set up the webhook secret in your Medplum project. Per the photon docs, if there is no secret, an empty string should be used.
-  const PHOTON_WEBHOOK_SECRET = event.secrets['PHOTON_ORDER_WEBHOOK_SECRET']?.valueString ?? '';
+  // const PHOTON_WEBHOOK_SECRET = event.secrets['PHOTON_ORDER_WEBHOOK_SECRET']?.valueString ?? '';
 
-  // Ensure the webhook is coming from Photon
-  const isValid = verifyEvent(webhook, PHOTON_WEBHOOK_SECRET);
-  if (!isValid) {
-    throw new Error('Not a valid Photon Webhook Event');
-  }
+  // // Ensure the webhook is coming from Photon
+  // const isValid = verifyEvent(webhook, PHOTON_WEBHOOK_SECRET);
+  // if (!isValid) {
+  //   throw new Error('Not a valid Photon Webhook Event');
+  // }
```

Comment:
I just commented this out for now. Is there a way to get the webhook header when executing the bot? If not, should I just remove this?